modules/org/infinispan/runtime/added/bin/launch.sh

```diff
@@ -75,16 +84,19 @@ generate_content
 
 # IDENTITIES_BATCH will not be set if IDENTITIES_PATH provided
 if [[ -n ${IDENTITIES_BATCH} ]]; then
-  # Use the native CLI if present
-  if [[ -f "${ISPN_HOME}/cli" ]]; then
-    ${ISPN_HOME}/cli --file ${IDENTITIES_BATCH}
-  else
-    ${ISPN_HOME}/bin/cli.sh --file ${IDENTITIES_BATCH}
+  execute_cli --file "${IDENTITIES_BATCH}"
+fi
+
+if [[ -n ${SERVER_LIBS} ]]; then
+  execute_cli install --server-home="${ISPN_HOME}" "${SERVER_LIBS}"
```

Comment:
We need to use the `-r` flag here, or provide a way for it to be explicitly configured on the image. IMO having a sensible default of `5` retries is sufficient for the Operator use-case.

Reply:
We also need `--clean`